src/connector.ts

```diff
@@ -668,27 +659,29 @@ async function standaloneConnect(
 export default {
   connectNonProjectREPLCommand: async (context: vscode.ExtensionContext) => {
     status.updateNeedReplUi(true);
+    let projectDirUri = state.getProjectRootUri();
+    if (!projectDirUri) {
+      projectDirUri = await state.getOrCreateNonProjectRoot(context, true);
+    }
+    await state.initProjectDir(projectDirUri);
```

Comment:
We add a `.catch` to this function in `connectCommand`, do we need one here?

Reply:
Iirc i added the one in `connectCommand` because the user can do funny stuff with the menu. Here we don't allow that. But it could also go wrong because of non-validated glob patterns, so yes, we should have one here anyway. =)